autoPyTorch/pipeline/components/training/losses.py

```diff
@@ -0,0 +1,59 @@
+from typing import Any, Dict, Optional, Type
+
+from torch.nn.modules.loss import (
+    CrossEntropyLoss,
+    BCEWithLogitsLoss,
+    MSELoss,
+    L1Loss
+)
+from torch.nn.modules.loss import _Loss as Loss
+
+
+losses = dict(classification=dict(
+    CrossEntropyLoss=dict(
+        module=CrossEntropyLoss, supported_output_type='multi-class'),
+    BCEWithLogitsLoss=dict(
+        module=BCEWithLogitsLoss, supported_output_type='binary-class')),
+    regression=dict(
+        MSELoss=dict(
+            module=MSELoss, supported_output_type='continuous'),
+        L1Loss=dict(
+            module=L1Loss, supported_output_type='continuous')))
+
+default_losses = dict(classification=CrossEntropyLoss, regression=MSELoss)
```

Comment:
I think this will change, so that binary classification is a task. That means, we are gonna need a default loss for this case.

Reply:
... which should also be CE then

Reply:
I have made it in a way that all types of classification have crossentropy as the default and all types of regression have MSE,  so what would happen is the task type comming from dataset properties would be split, [like here](https://github.com/ravinkohli/Auto-PyTorch_refactor/blob/e1182883ddc8806bd98b2e93c12aebbe3df80f7a/autoPyTorch/pipeline/components/training/losses.py#L27).